app/lib/widgets/wallets/send_confirmation.dart

```diff
@@ -164,6 +164,8 @@ class _SendConfirmationWidgetState extends State<SendConfirmationWidget> {
         loading = false;
       });
       return;
+    } finally {
+      Navigator.pop(context);

```

Comment:
in the failure case, we shouldn't go back

Reply:
Fixed.